backend/src/journal/entities/journal.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+
+@ObjectType()
+export class Journal {
+  @Field(() => String)
+  id: string;
+
+  @Field(() => String)
+  userId: string;
+
+  @Field(() => String)
+  title: string;
+
+  @Field(() => String)
+  content: string;
```

Comment:
In your prisma.schema `content` is optional.
If there can be a case where you create a journal with an empty content — this journal graphql entity will throw an error

Either make this content field on graphql entity nullable, or make the content required at the level of DB

Reply:
Done!